content/pages/meetup/2023/0920-nycu.rst

```diff
@@ -0,0 +1,105 @@
+========================================
+Meetup 2023 September 20th in NYCU
+========================================
+
+:date: 2023-08-31 13:00
+:url: meetup/2023/0920-nycu
+:save_as: meetup/2023/0920-nycu.html
+
+(DRAFT)
+
+Date
+-----
+
+* Date: September 20th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 19:00―19:10 presentation: what are the things
+* 19:10―19:30 presentation: how to get things done
+* 19:30―19:45 discussion: things you have done
+* 19:45―20:00 case study: sciwork meetup
+* 20:00―20:15 discussion: analyze the things you want to do
+* 20:15―20:30 hands-on: make the plan and review
+
+|
+
+Subjects
+------------------
+
+Training: how to work out unknown things
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Speaker: 
+Yung-Yu - Engineer from Synopsis and enthusiastist of numerical code. He is also 
+the leader and founder of PyCon and sciwork.
+
+Training assistants: 
+Chun-Hsu - WiFi firmware engineer from Mediatek.Inc and developer of `modmesh <https://github.com/solvcon/modmesh>`_. 
+Chester - Software engineer in test from SiFive and developer of `swportal <https://github.com/sciwork/swportal>`_ 
+and `swconf <https://github.com/sciwork/swconf>`_.
+Steve - SDK engineer from Clounix and volunteer of `sciwork <https://sciwork.dev/>`_.
+
+In this training, Yung-Yu will talk about practical practices that are applied to general 
+scenario, from your research to your career. You will be trained on how to work 
+out unknown things and improve your skills in framing problems and conducting these tasks.
+
+Preparational meeting for September scisprint
```

Comment:
Please add time slots after the do-unknown-things for the "preparative" meeting.

And there is no such word of "preparational".  Please correct it.